spec/support/controller_macros.rb

```diff
@@ -1,7 +1,7 @@
 # Taken unashamedly from https://github.com/plataformatec/devise/wiki/How-To%3a-Controllers-and-Views-tests-with-Rails-3-%28and-rspec%29
 module ControllerMacros
   def login_member(member_factory=:member)
-    before(:each) do
+    before do
       @request.env["devise.mapping"] = Devise.mappings[:member]
       member = FactoryGirl.create(member_factory || :member)
```

Comment:
I think this line can be replaced with a `let`, but I think the `@request.env` line probably has to stay as it is.